config/StoredProcedures/GetAllInventory.sql

```diff
@@ -30,4 +31,4 @@ on s1.ProductID = s2.ProductID;
 
 END $$
 
-DELIMITER ;
\ No newline at end of file
+DELIMITER ;
```

Comment:
This one might work:
SELECT S1.ProductID, S1.ProductName, IFNULL(SUM(S1.QuantityAvailable), 0) AS TotalQuantityAvailable, S1.DateCreated AS LastRunDate, S1.RunID AS LastRunID, S1.InitialQuantity AS LastRunInitialQuantity
FROM
	  (Select Pr.ProductID, Pr.Name AS ProductName, R.RunID, R.DateCreated, R.InitialQuantity, R.QuantityAvailable
	  FROM Products Pr LEFT JOIN Piles Pi ON Pr.ProductID = Pi.ProductID LEFT JOIN Runs R ON Pi.PileID = R.PileID
	  WHERE Pr.ViewOption = 1
      ORDER BY Pr.ProductID ASC, R.DateCreated DESC, R.RunID DESC) AS S1
GROUP BY S1.ProductID;